packages/app-project/src/shared/components/ProjectStatistics/components/Subtitle/Subtitle.js

```diff
@@ -1,10 +1,11 @@
 import { Text } from 'grommet'
+import { withResponsiveContext } from '@zooniverse/react-components'
 import { object, oneOfType, string } from 'prop-types'
 import React from 'react'
 
-function Subtitle ({ mode, text, ...props }) {
+function Subtitle ({ margin, mode, screenSize, text, ...props }) {
   return (
-    <Text {...props}>
+    <Text {...props} margin={(screenSize === 'small') ? { top: 'none', bottom: 'xsmall' } : margin}>
```

Comment:
I'd recommend moving this ternary out of the JSX for legibility, as you've done later on